.gitignore

```diff
@@ -90,10 +90,15 @@ target/
 .mypy_cache/
 
 # model checkpoints
-/models
+models/
```

Comment:
This need to be `/models` or `/models/` (maybe `.models/` also work?). Otherwise `src/models/` also gets ignored!

Reply:
In other words `models/` ignores `src/models` which we don't want.